metadata-ingestion/archived/source_docs/snowflake.md

```diff
@@ -170,7 +170,7 @@ Note that a `.` is used to denote nested fields in the YAML recipe.
 
```

Comment:
Docs have moved into source code.  Please see latest snowflake config on master and merge the changes in.

Reply:
done